independent-projects/bootstrap/maven-resolver/src/test/java/io/quarkus/bootstrap/workspace/test/LocalWorkspaceDiscoveryTest.java

```diff
@@ -128,8 +127,6 @@ public void loadIndependentProjectInTheWorkspaceTree() throws Exception {
         final Map<AppArtifactKey, LocalProject> projects = project.getWorkspace().getProjects();
         assertEquals(6, projects.size());
         assertTrue(projects.containsKey(new AppArtifactKey(MvnProjectBuilder.DEFAULT_GROUP_ID, "independent")));
-
-        assertLocalDeps(project, "root-module-not-direct-child", "root-no-parent-module", "root-module-with-parent");
```

Comment:
Can you please explain the rationale around the removal of these assertions?

Reply:
I removed LocalProject.getSelfWithLocalDeps() method because it did not return a complete set of dependencies and was only used in DevMojo.

Reply:
Gotcha, thanks